rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1231,6 +1233,24 @@ public long getEstimatedFeesForNextPegOutEvent(Object[] args) throws IOException
         return bridgeSupport.getEstimatedFeesForNextPegOutEvent().value;
     }
 
+    public byte[] getPegoutCreationRskTxHashByBtcTxHash(Object[] args) {
+        logger.trace("getPegoutCreationRskTxHashByBtcTxHash");
+        try {
+            byte[] hashBytes = (byte[])args[0];
+            Sha256Hash hash = Sha256Hash.wrap(hashBytes);
+
+            Optional<Keccak256> rskTxHash = this.bridgeSupport.getPegoutCreationRskTxHashByBtcTxHash(hash);
+            if (rskTxHash.isPresent()){
+                return rskTxHash.get().getBytes();
+            } else {
+                return ByteUtil.EMPTY_BYTE_ARRAY;
+            }
+        } catch (Exception e) {
```

Comment:
Is there any exception thrown in any of the methods called?

Reply:
No, there is not any exception thrown intentionally. I put it thinking on runtime exceptions but looking at the other methods those methods don't have a try/catch, so it may be removed and let the caller handle the exception if there is any.